src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -219,7 +219,7 @@ const Conversation: React.FC<ConversationProps> = props => {
     <NoScrollFlex flexDirection="column" width="100%">
       <MessageContainer>
         <Box>
-          <Spacer mt={["75px", 2]} />
+          <Spacer mt={["75px", "75px", 2]} />
```

Comment:
additional margin top for first message in the conversation


Before:
<img width="811" alt="Screen Shot 2020-07-21 at 12 31 24 PM" src="https://user-images.githubusercontent.com/12748344/88084392-5c07e700-cb52-11ea-8bd8-3bc5b921b809.png">
<br>
<br>

After:
<img width="781" alt="Screen Shot 2020-07-21 at 12 28 45 PM" src="https://user-images.githubusercontent.com/12748344/88084421-6b873000-cb52-11ea-92ed-e6c86c90e627.png">